File: data/src/main/java/com/joxad/zikobot/data/module/soundcloud/model/SoundCloudUser.java
package com.joxad.zikobot.data.module.soundcloud.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Created by josh on 15/06/16.
 */
@Parcel
@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class SoundCloudUser {


    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("kind")
    @Expose
    public String kind;
    @SerializedName("permalink")
    @Expose
    public String permalink;
    @SerializedName("username")
    @Expose
    public String username;
    @SerializedName("uri")
    @Expose
    public String uri;
    @SerializedName("permalink_url")
    @Expose
    public String permalinkUrl;
    @SerializedName("avatar_url")
    @Expose
    public String avatarUrl;
    public SoundCloudUser() {
    }
}

File: data/src/main/java/com/joxad/zikobot/data/module/soundcloud/model/SoundCloudUser.kt
package com.joxad.zikobot.data.module.soundcloud.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


import lombok.AllArgsConstructor
import lombok.Data

/**
 * Created by josh on 15/06/16.
 */
class SoundCloudUser {


    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("kind")
    @Expose
    var kind: String? = null
    @SerializedName("permalink")
    @Expose
    var permalink: String? = null
    @SerializedName("username")
    @Expose
    var username: String? = null
    @SerializedName("uri")
    @Expose
    var uri: String? = null
    @SerializedName("permalink_url")
    @Expose
    var permalinkUrl: String? = null
    @SerializedName("avatar_url")
    @Expose
    var avatarUrl: String? = null
}
